src/render/batch.rs

```diff
@@ -236,66 +237,113 @@ impl<T: Clone + PartialEq> Array<T> {
 }
 
 
-/// Ref batch - copyable and smaller, but depends on the `Context`.
+/// Referrenced core - a minimal sealed batch that depends on `Context`.
 /// It has references to the resources (mesh, program, state), that are held
 /// by the context that created the batch, so these have to be used together.
-pub struct RefBatch<T: ShaderParam> {
+pub struct RefCore<T: ShaderParam> {
```

Comment:
I haven't really thought about names, no. `RefBatchData` doesn't sound too good because it bears the "data" term, while in fact `RefCore` can be seen as a batch type. Yes it is incomplete (requires not only the context, but also a slice and a parameter data), but neither is `RefBatch` (needs the context).

I don't insist on `RefCore` name though. My understanding is that since we don't re-export everything in `batch` module, this type is accessed as `gfx::batch::RefCore`, which makes it less ambiguous. Perhaps, you'd prefer `CoreBatch` more?